pgrx-tests/tests/ui/invalid_pgcast_function.rs

```diff
@@ -0,0 +1,8 @@
+use pgrx::prelude::*;
+
+#[pg_cast]
+pub fn cast_function() -> i32 {
+    0
+}
```

Comment:
Removed this one because this is a runtime error that only get triggered when we actually generate some SQL entities. Therefore it does not fit as a ui test.

Reply:
Oh okay! I sometimes don't remember what actually happens during compilation and what is technically """runtime""". I'd like to add schema-gen testing to the UI work, but I think that may need a burlier library than trybuild to do it. Thanks!